dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -115,6 +115,16 @@ public void testGitHubReleaseNoWorkflowOnDockstore() throws Exception {
         assertEquals("Should be a CWL workflow", Workflow.DescriptorTypeEnum.CWL, workflow2.getDescriptorType());
         assertEquals("Should be type DOCKSTORE_YML", Workflow.ModeEnum.DOCKSTORE_YML, workflow2.getMode());
         assertEquals("Should have one version 0.2", 1, workflow2.getWorkflowVersions().size());
+
+        // Branch master on GitHub - updates two existing workflows
+        workflows = client.handleGitHubRelease(workflowRepo, "DockstoreTestUser2", "refs/heads/master", installationId);
+        assertEquals("Should only have two services", 2, workflows.size());
+
+        workflow = client.getWorkflowByPath("github.com/" + workflowRepo + "/foobar", "", false);
+        assertEquals("Should have three versions 0.1, 0.2, and master", 3, workflow.getWorkflowVersions().size());
```

Comment:
Ditto

Reply:
In this case I am explicitly adding versions, not calling a refresh. But I'll update to check for versions by name.